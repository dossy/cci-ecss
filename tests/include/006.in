@pragma set_include_path("tests/include/x:tests/include/x/y");
@include "006-1.inc";
@include "006-2.inc";
