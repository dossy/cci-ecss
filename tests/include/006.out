.abc{baz:blah;}
.xyz{foo:bar;}
